cldfbench_grambank.py

```diff
@@ -185,6 +186,15 @@ def cmd_makecldf(self, args):
         print('selecting best sheets')
         sheets = list(iterunique(sheets, verbose=True))
 
+        # Add values from conflict sheets
```

Comment:
I struggle to evaluate this, we should probably just leave it to @xrotwang . I'd expect to see something about the columns `Conflict` and the column `Select`, but that specification is probable somewhere else.